src/com/epam/Exceptions/helpers/UniversityHelper.java

```diff
@@ -0,0 +1,248 @@
+package com.epam.Exceptions.helpers;
+
+import com.epam.Exceptions.FacultyAbsenceException;
+import com.epam.Exceptions.GroupAbsenceException;
+import com.epam.Exceptions.StudentAbsenceException;
+import com.epam.Exceptions.SubjectAbsenceException;
+import com.epam.Exceptions.models.*;
+
+import java.util.Random;
+
+public class UniversityHelper {
+
+    public static University creteTheUniversity() {
+        String mathematics = "Mathematics";
```

Comment:
It is a good practice to create constants in these cases. For example: 
private static final String MATHEMATICS = "Mathematics";